test/core/annotations/test_annot_throws_exception.pkb

```diff
@@ -0,0 +1,76 @@
+create or replace package body test_annot_throws_exception
```

Comment:
Can you add negative scenario tests for:
--%throws - with no parameters
--%throws( some bad and dummy value )
--%throws(-1234,+1234,some bad value)
--%throws(, .. , )
So we know what it does exactly

Reply:
I'll add them

Reply:
This looks great!